src/main/steps/applicant/yourdocuments/alldocuments/yourwitnessstatements/content.ts

```diff
@@ -49,7 +49,7 @@ export const generateContent: TranslationFn = content => {
   const translations = languages[content.language]();
   const orders: object[] = [];
   for (const doc of content.userCase?.citizenUploadedDocumentList || []) {
-    if (doc.value.documentType === 'Your witness statements' && doc.value.isApplicant === content.byApplicant) {
+    if (doc.value.documentType === 'Your witness statements' && doc.value.partyName === content.name) {
```

Comment:
@SasankAchyuta9 It's better to define a const for 'Your witness statements' in one place & use in all places for comparison. If any changes in future we can just change the const.

Reply:
Yes , This is taken care of patil.